atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/CassandraSchemaLockTest.java

```diff
@@ -52,11 +53,14 @@
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 public class CassandraSchemaLockTest {
-    private static final int THREAD_COUNT = 4;
+
+    private static String cassandraVersion = System.getenv("CASSANDRA_VERSION");
```

Comment:
Why don't we put the `getenv` call inside `ThreeNodeCassandraCluster`, so we don't have to have this in every test that uses it?

Also would suggest putting a default value here - currently if the env var is not set then you'd get null, and presumably terrible things would happen? Thinking particularly of not breaking the workflow of running through IntelliJ.